### PR TITLE
Issue#1107 TableAdmin samples splited into data_admin_snippets

### PR DESCRIPTION
**Pull request by roopak-qlogic**
Fixes issue #1107

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-cpp/1167)

### CODE REVIEW

File: google/cloud/bigtable/examples/data_admin_snippets.cc [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+  auto last_slash = std::string(cmd).find_last_of('/');
+  auto program = cmd.substr(last_slash + 1);
+  std::cerr << msg << "\nUsage: " << program << " <command> [arguments]\n\n"
+            << "Commands:\n"
+            << command_usage << std::endl;
+}
+
+void Apply(google::cloud::bigtable::Table table, int argc, char* argv[]) {
+  if (argc != 1) {
+    throw Usage{"apply: <project-id> <instanse-id> <table-id>"};
```
Comment (coryan) [inline google/cloud/bigtable/examples/data_admin_snippets.cc]: here and everywhere else s/instanse/instance/.

File: google/cloud/bigtable/examples/BUILD [hunk 2]
```diff
@@ -56,4 +56,9 @@ cc_binary(
     name = "table_admin_snippets",
     srcs = ["table_admin_snippets.cc"],
     deps = ["//google/cloud/bigtable:bigtable_client"],
+
```
Comment (coryan) [inline google/cloud/bigtable/examples/BUILD]: You need a closing parenthesis `)` here.

**Comment by roopak-qlogic** [comment]
Issue#1059 and Issue#1057 is also completed, will push for review, Once it is complete.

**Comment by roopak-qlogic** [comment]
PTAL

**Pull request merged**